src/js/modules/infragistics.ui.editors.js

```diff
@@ -12059,5 +12062,687 @@
 			}
 		}
 	});
+	$.widget("ui.igTimePicker", $.ui.igDateEditor, {
+		options: {
+			/* type="object" Gets/Sets delta-value which is used to generate the drop-down items for the time picker.
+				Accepted values for deltas are positive integer numbers, and the fractional portion of floating point numbers is ignored.
+			```
+				//Initialize with object
+				$(".selector").%%WidgetName%%({
+					timeDelta: {
+						hours: 0,
+						minutes: 30,
+					}
+				});
+
+				//Get
+				var timeDelta= $(".selector").%%WidgetName%%("option", "timeDelta");
+
+				//Set with object
+				$(".selector").%%WidgetName%%("option", "timeDelta", { minutes: 30 });
+			```
+			object type="object" A configuration object, which defines specific values for each time period. The option can accept the following format:
+				timeDelta: {
+					hours: 0,
+					minutes: 30,
+				}
+			Time periods that don't have values use 0 as default for hours and 30 for minutes.
+			*/
+			itemsDelta: { hours: 0, minutes: 30 },

```

Comment:
the option is called itemsDelta, while the code snippets are showing timeDelta. either update code snippet, or add the corresponding property.

Reply:
I have changed the code snippet